src/zenml/zen_server/deploy/helm/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-name: zenml
-version: "1.0"
+name: zenml-server
```

Comment:
No strong reason. I felt that this reflected what the helm chart deploys, more appropriately, since it is now in the public light more than ever. Let me know if I should consider going back

Reply:
No need to go back, just considering the implications in the name change. No idea what this does for current deployments.